### ISSUE TITLE
activity logs

### ISSUE THREAD

**Issue by nthieberger** [paradisec-archive/nabu#405]
We currently get access to recent activity logs but is it possible to have all activity logs generated on request? For example, I need to track who created a particular collection and this will need to be found  earlier than the present logs provide.

**Comment by enwardy** [comment]
I also have encountered a few instances where it would be helpful to be able to generate an activity log on request

**Comment by johnf** [comment]
@nthieberger Would this still be useful. I'm thinking we add an activity log link for admins on items and collections which show the activities for those items

**Comment by nthieberger** [comment]
Yes, if it is not too much work to expose the list of all activity , maybe as a download?
